package.json

```diff
@@ -87,5 +87,8 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "jest": {
+    "testEnvironment": "node"
```

Comment:
Optional: I _think_ I'd prefer to have the Jest config in it's own file. Doing so is pretty standard at this point and harder to miss than tucking it away in the `package.json`